db/migrate/20150422201652_add_approved_to_companies.rb

```diff
@@ -0,0 +1,5 @@
+class AddApprovedToCompanies < ActiveRecord::Migration
+  def change
+    add_column :companies, :status, :string
+  end
+end
```

Comment:
Also, since there are going to be only a few different options for status, could we add a validation into the company model that the status is included in an array of choices. Ex ["contacted", "uncontacted", "confirmed", "dead"]